docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -234,7 +265,6 @@ The following services are disabled by default:
 - support-notifications
 - support-scheduler
 - sys-mgmt-agent - *deprecated EdgeX component*
-- device-virtual
```

Comment:
- device-virtual - *deprecated; use the standalone [EdgeX device virtual snap](#edgex-device-virtual)*

Reply:
It is not deprecated. It has been fully removed.